.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please be sure that all of your linters are passing before asking for your next review 😉 Stylelint checks are failing 👀

Reply:
Addressed ✔